recipe/meta.yaml

```diff
@@ -38,12 +40,19 @@ test:
     - 7zr a xxx.7z xxx
 
 about:
-  home: http://sourceforge.net/projects/p7zip/
-  summary: p7zip is a quick port of 7z.exe and 7za.exe (command line version of 7zip, see www.7-zip.org ) for Unix.
+  home: https://sourceforge.net/projects/p7zip/
+  summary: p7zip is a port of the Windows programs 7z.exe and 7za.exe provided by 7-zip.
+  description: |
+    7-zip is a file archiver with the highest compression ratio. See
+    for more information www.7-zip.org site.
   license: LGPL-2.0-or-later
```

Comment:
It has the `unRAR` restriction:
```
  Licenses for files are:

    1) CPP/7zip/Compress/Rar* files:  GNU LGPL + unRAR restriction
    2) All other files:  GNU LGPL

  The GNU LGPL + unRAR restriction means that you must follow both 
  GNU LGPL rules and unRAR restriction rules.
```
But it's not spdx compliant.

Reply:
should be address